src/pages/dev/amplifier/chain-integration/introduction.mdx

```diff
@@ -1,63 +1,101 @@
+
+- [Source chain gateway](/dev/amplifier/chain-integration/introduction#source-chain-gateway-contract)
+- [Axelar gateway](/dev/amplifier/chain-integration/introduction#axelar-gateway-contract)
+- [Voting verifier](/dev/amplifier/chain-integration/introduction#verifier-contract)
+- [Multisig prover](/dev/amplifier/chain-integration/introduction#prover-contract)
+
+### Message relayer
+
+- This is a service built with any tech stack that listens for events on your chain and relays them to the Axelar network.
+- For a detailed implementation, see the [Message Relayer example](https://github.com/axelarnetwork/axelar-examples/tree/main/examples/amplifier) on GitHub.
+
+### Source chain gateway contract
+
+- This is the gateway contract for your source chain, built using the same tech stack as that chain. It serves as the primary API for developers sending messages from your chain.
+- The contract must implement the [Cross-Chain Gateway Protocol](https://github.com/axelarnetwork/cgp-spec).
+- EVM chains can re-use the existing [EVM Gateway](https://github.com/axelarnetwork/axelar-cgp-solidity).
+
+### Axelar gateway contract
+
+- This is a smart contract on the Axelar network that checks the validity of a transaction with the verifier contract and forwards it to the Amplifier router.
+- Most integrators will begin by using or customizing the Axelar [Gateway contract](https://github.com/axelarnetwork/axelar-amplifier/tree/main/contracts/gateway) (current version: 0.2.0).
+- [Instantiation parameters](https://github.com/axelarnetwork/axelar-amplifier/blob/main/contracts/gateway/src/msg.rs) can be found on GitHub.
+
+### Voting verifier contract
+
+- This is a smart contract on the Axelar network that verifies the validity of transactions on your chain.
+- Most integrators will begin by using or customizing the Axelar [Voting Verifier contract](https://github.com/axelarnetwork/axelar-amplifier/tree/main/contracts/voting-verifier) (current version: 0.3.0), which can be customized to your needs for cases such as verification via ZK Proof.
+- [Instantiation parameters](https://github.com/axelarnetwork/axelar-amplifier/blob/main/contracts/voting-verifier/src/msg.rs) can be found on GitHub.
+- If you deploy a Voting Verifier contract, verifiers will need to support your chain and vote on the truth of source chain transactions.
+
+### Multisig prover contract
+
+- This is a smart contract on the Axelar network that knows how to prove the validity of a transaction on your chain.
+- Most integrators will begin by using or customizing the Axelar [Multisig Prover contract](https://github.com/axelarnetwork/axelar-amplifier/tree/main/contracts/multisig-prover) (current version: 0.3.0).
+- [Instantiation parameters](https://github.com/axelarnetwork/axelar-amplifier/blob/main/contracts/multisig-prover/src/msg.rs) can be found on GitHub.
```

Comment:
I think the purpose of this page is a conceptual overview, not an implementation guide, so I think the specific parameters gets in the way of understanding.